src/EhrgoHealth.WebService/Models/AllergyIntolerance.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Rest;
+
+namespace EhrgoHealth.WebService.Models
+{
+    public class AllergyIntolerance
+    {
+        //Dictonary looks up possible allergies for a given medication. Example allergyLookup<Medication, List<Codes>>
+        private Dictionary<string, List<string>> allergyLookup = new Dictionary<string, List<string>>();
+        private FhirClient fhirClient = null;
+
+
+        public AllergyIntolerance(string fhirServer)
+        {
+            if (fhirServer == null || fhirServer == String.Empty)
+                throw new Exception("Invalid URL passed to AllergyIntolerance Constructor");
+
+            fhirClient = new FhirClient(fhirServer);
+
+            //We do not use a database for the webservice. We will hardcode a handful of allergies and their intolerances
+            allergyLookup.Add("hydrocodone", new List<string>() { "Z88.5" });
+        }
+
+        //The controller is expected to receive two pieces of data from an EHR.
+        //The List of medications they currently know their patient is taking
+        //The patient's ID from FHIR
+        /// <summary>
+        /// The controller is expected to receive two data structures from an EHR.
+        /// </summary>       
+        /// <param name="patientID"> The patient's ID from FHIR</param>
+        /// <param name="medications"> The List of medications they currently know their patient is taking</param>
+        public List<string> GetListOfMedicationAllergies(uint patientID, List<string> medications)

```

Comment:
I'm sure you can make your int's ints and not unsigned int's although I do not predict we will need negative numbers, I also do not predict us needing int's over 2 million